aspnetcore/grpc/migration.md

```diff
@@ -50,7 +50,7 @@ public void ConfigureServices(IServiceCollection services)
 {
     services.AddGrpc(options =>
     {
-        options.ReceiveMaxMessageSize = 16384; // 16 MB
+        options.ReceiveMaxMessageSize = 16 * 1024 * 1024; // 16 MB
```

Comment:
Since this changes to a non-default message size as a demonstration, I think a better fix is to just remove the `// 16 MB` in favor of `// Changes the default 16 MB limit`.

Reply:
AFAIK the default is 4MB.   So it's just that the comment and the actual value don't match.

Reply:
Yes, indeed ... so it is.

Reply:
Feel free to change to comment to  `// Changes the default limit to 16MB` or sth like that.

Reply:
It's cool. We link in-text better anyway. Those remarks get translated into other languages. Code comments are stuck in English forever.